### ISSUE TITLE
Stars are not coming inline (in row)

### ISSUE THREAD

**Issue by amitbravo** [fraserxu/ionic-rating#32]
on Ionic, stars are not coming a one row , I tried to change in ionic-rating.css ul.rating li { } but does not change anything, however, i can see there is already display: inline being used but it is not working .

**Comment by toxic2302** [comment]
same problem here +1

**Comment by vlafranca** [comment]
Hi,
You must have a css conflict somewhere, I already got this issue and it was because of a rule being overwritten somewhere (it was long time ago and I don't remeber unfortunately)
You can check this codepen as an example : http://codepen.io/vlafranca/pen/GoxVKL

I recommend you to inspect the stars with a debugger (chrome, firefox etc.) and look at the css rule being overwritten.

**Comment by toxic2302** [comment]
Hey, 
thanks for this little tutorial but it doesn't work. I'm using this rating tag in an `<form>` element between two `<input> `fields. I have added a css class with `display: inline-block` but it doesn't change anything.

Do you have some more tricks for me? :smile:

**Comment by vlafranca** [comment]
Can you make a codepen please ?

**Comment by toxic2302** [comment]
Codepen is created and it works but not in my project :(

**Comment by vlafranca** [comment]
So the error comes from your project ^^ you have to debug ...

**Comment by toxic2302** [comment]
Yes I have to debug my Project. I think it crashed with the materialize css. I have tested this in a diffrent codepen. http://codepen.io/toxic2302/pen/VjLmeG

**Comment by ciroanacleto** [comment]
I have the same issue.
I'm using  the component inside a `<li>` element with ng-repeat. For example:
```html
<ul class="list">
    <li ng-repeat="comment in dish.comments | orderBy:orderText">
        <blockquote>
            <rating readonly="true" ng-model="comment.rating" max="maxRating"></rating>
            <p>{{comment.comment}}</p>
        </blockquote>
    </li>
</ul>
```
If the css class it's being overwritten must be by the ionic css classes. How I can solve this ?

**Comment by ciroanacleto** [comment]
Hi, as @vlafranca said, there were a css class overwriting my <li> tag and so the stars not comming in line. I copied the directive css class and add in my project css file. This way worked fine.

**Comment by toxic2302** [comment]
Thank you @ciroanacleto. I have copied the css classes from ionic-rating in my own style.css and now it works. ;)

**Comment by ciroanacleto** [comment]
@toxic2302 There are a better way. Import the directive's css in your index.html, like this :
```html 
<link href="lib/ionic-rating/ionic-rating.css" rel="stylesheet">
```
Will work too.

**Comment by wasa4587** [comment]
`<link href="lib/ionic-rating/ionic-rating.css" rel="stylesheet">`
Worked for me
